src/Components/Aspire.Microsoft.Azure.Cosmos/Aspire.Microsoft.Azure.Cosmos.csproj

```diff
@@ -21,6 +21,9 @@
     <PackageReference Include="Microsoft.Extensions.Configuration.Binder" />
     <PackageReference Include="Microsoft.Extensions.Hosting.Abstractions" />
     <PackageReference Include="OpenTelemetry.Extensions.Hosting" />
+
+    <!-- Pinned version for Component Governance - Remove when root dependencies are updated tracked by https://github.com/Azure/azure-cosmos-dotnet-v3/issues/4302 -->
```

Comment:
1. From reading the linked issues, it sounds like this will never be fixed. Does a comment to remove this add value?
2. Maybe we should link to the root issue instead of a dupe-of-a-dupe-of-a-dupe.